bin/build

```diff
@@ -10,16 +10,24 @@ MIQ_ORG=${MIQ_ORG:-ManageIQ}
 SUI_ORG=${SUI_ORG:-ManageIQ}
 APPLIANCE_ORG=${APPLIANCE_ORG:-ManageIQ}
 
+REPO_PREFIX=${REPO_PREFIX:-manageiq}
```

Comment:
I think a better name for this would be `CORE_REPO_NAME` in the Dockerfile it would make more sense to see `$CORE_REPO_NAME-appliance` rather than an entire repo name that consists of a "prefix".